draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -11087,6 +11087,20 @@ Content-Encoding: gzip
 </t>
 </section>
 
+<section title="Attacks using Compression to Violate Confidentiality" anchor="compression.attacks">
+<t>
+   Some attacks on encrypted protocols use the differences in size created by compression
+   to reveal confidential information; for example, <xref target="BREACH"/>.
+</t>
+<t>
+   HTTPS messages can be compressed in a number of ways, including using TLS compression,
+   content-codings, transfer-codings, and other extension or version-specific mechanisms.
```

Comment:
This is where you probably want to nod toward HPACK and its security considerations.  Some of these compression schemes are believed to be insufficiently general and so are not exploitable, though care is still needed.